install/resources/mybb_theme.xml

```diff
@@ -4928,6 +4928,21 @@ if(use_xmlhttprequest == "1")
 </td>
 </tr>
 </table>
+<br />]]></template>
+		<template name="member_profile_adminoptions_manageban" version="1824"><![CDATA[<table border="0" cellspacing="{$theme['borderwidth']}" cellpadding="{$theme['tablespace']}" width="100%" class="tborder">
+<tr>
+<td colspan="2" class="thead"><strong>{$lang->admin_options}</strong></td>
+</tr>
+<tr>
+<td class="trow1">
+<ul>
+<li><a href="{$mybb->settings['bburl']}/{$admin_dir}/index.php?module=user-users&amp;action=edit&amp;uid={$uid}">{$lang->admin_edit_in_acp}</a></li>
+<li><a href="{$mybb->settings['bburl']}/{$admin_dir}/index.php?module=user-banning&amp;action=edit&amp;uid={$uid}">{$lang->admin_edit_ban_in_acp}</a></li>
+<li><a href="{$mybb->settings['bburl']}/{$admin_dir}/index.php?module=user-banning&amp;action=lift&amp;uid={$uid}&amp;my_post_key={$mybb->post_code}">{$lang->admin_lift_ban_in_acp}</a></li>
```

Comment:
```suggestion
<li><a href="{$mybb->settings['bburl']}/{$admin_dir}/index.php?module=user-banning&amp;action=lift&amp;uid={$uid}">{$lang->admin_lift_ban_in_acp}</a></li>
```

We don't need this because the Admin CP just shows a confirmation form and doesn't rely on the post key.